app/assets/stylesheets/mixins/_helpers.scss

```diff
@@ -6,5 +6,5 @@
 
 // https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/_shims.scss#L45
 .clearfix {
```

Comment:
Could we do a global search-and-replace in the templates instead of extending the `clearfix` class?

Reply:
I've just done the search, and there appears to be no instances of this class anywhere in the app! Odd. Guess I'll take it out completely then...